### PR TITLE
FFCP LCD support

### PR DESCRIPTION
**Pull request by fragginright**

### CODE REVIEW

File: klippy/extras/display/hd44780.py [hunk 1]
```diff
@@ -12,11 +12,21 @@
 
 class HD44780:
     def __init__(self, config):
+        self.isSrDisplay = False
```
Comment (etet100) [inline klippy/extras/display/hd44780.py]: This name suggests it's some kind of standard protocol. I don't think it is.

**Comment by KevinOConnor** [comment]
Thanks.  FYI, I went ahead and added an example config for the mightyboard (commit ff0324b0) without the display.

My high-level comments on this change is that it looks like the micro-controller code ultimately implements a software spi system.  I wonder if we can reuse the existing src/spi_software.c code.  The new code has a lot of `usdelay()` timing calls, but ultimately all these calls get compiled out as the usdelay function doesn't do anything on a 16Mhz atmega1280 machine.  So, without these timing calls, it may be possible to implement this entirely in the Python host code.

Also, commits need to have a signed-off-by line - see: https://github.com/KevinOConnor/klipper/blob/master/docs/CONTRIBUTING.md

-Kevin

**Comment by KevinOConnor** [comment]
Any updates on this?

-Kevin

**Comment by KevinOConnor** [comment]
Okay, I'm going to mark this as inactive.

-Kevin

**Pull request closed without merging**